packages/components/app/styles/components/side-nav/main.scss

```diff
@@ -66,56 +72,100 @@
 
 .hds-side-nav__menu-toggle-button {
   position: absolute;
+  top: calc(var(--hds-app-sidenav-border-width) * -1);
+  left: calc(var(--hds-app-sidenav-border-width) * -1);
   z-index: 1;
+  display: block;
+  width: 26px;
+  height: 36px;
+  padding: 9px 4px;
   color: var(--token-color-foreground-high-contrast);
+  text-align: right;
   background: none;
-  border: 1px solid transparent;
+  background-color: var(--token-side-nav-color-surface-primary);
+  border: var(--hds-app-sidenav-border-width) solid var(--hds-app-sidenav-border-color);
+  border-left-color: transparent;
+  border-top-right-radius: var(--hds-app-sidenav-border-radius);
+  border-bottom-right-radius: var(--hds-app-sidenav-border-radius);
+  transform:
+    translateX(var(--hds-app-sidenav-width-expanded))
+    translateY(24px);
   cursor: pointer;
-  transition-timing-function: var(--hds-app-sidenav-animation-easing);
-  transition-duration: var(--hds-app-sidenav-animation-duration);
-  transition-property: width, height, padding, border-radius, transform;
+  transition:
+    transform var(--hds-app-sidenav-animation-duration)
+    var(--hds-app-sidenav-animation-easing),
+    width var(--hds-app-sidenav-animation-duration)
```

Comment:
I know I'm starting to split hairs here, but I think the animation of the hover state could be tightened up a bit. Rather than the same duration value as the entire collapse animation, can this duration be 150ms? That seems like a happy medium given the size and impact.

Reply:
Sure, let's give that a go and see how it feels